src/components/Practice3.tsx

```diff
@@ -0,0 +1,9 @@
+import { InputField } from './InputField';
+
+export const Practice3 = () => {
+  return (
+    <>
```

Comment:
You don't need Fragments (`<>`) when returning only one element in the component!